vhdbuilder/packer/vhd-image-builder-mariner-gen2.json

```diff
@@ -324,9 +338,26 @@
     {
       "type": "shell",
       "inline": [
+        "sudo FEATURE_FLAGS={{user `feature_flags`}} BUILD_NUMBER={{user `build_number`}} BUILD_ID={{user `build_id`}} COMMIT={{user `commit`}} HYPERV_GENERATION={{user `hyperv_generation`}} CONTAINER_RUNTIME={{user `container_runtime`}} TELEPORTD_PLUGIN_DOWNLOAD_URL={{user `teleportd_plugin_download_url`}} ENABLE_FIPS={{user `enable_fips`}} SGX_INSTALL={{user `sgx_install`}} /bin/bash -ux /home/packer/pre-install-dependencies.sh",
```

Comment:
shouldn't there be a 2nd reboot between these for fips? or am I missing something?

Reply:
ah, touche